src/test/java/easytests/support/UsersSupport.java

```diff
@@ -0,0 +1,230 @@
+                (Boolean) fixture[6],
+                (Integer) fixture[7]
+        );
+    }
+
+    public UserModelInterface getModelMock(
+            Integer id,
+            String firstName,
+            String lastName,
+            String surname,
+            String email,
+            String password,
+            Boolean isAdmin,
+            Integer state
+    ) {
+        final UserModelInterface userModel = Mockito.mock(UserModelInterface.class);
+        Mockito.when(userModel.getId()).thenReturn(id);
+        Mockito.when(userModel.getFirstName()).thenReturn(firstName);
+        Mockito.when(userModel.getLastName()).thenReturn(lastName);
+        Mockito.when(userModel.getSurname()).thenReturn(surname);
+        Mockito.when(userModel.getEmail()).thenReturn(email);
+        Mockito.when(userModel.getPassword()).thenReturn(password);
+        Mockito.when(userModel.getIsAdmin()).thenReturn(isAdmin);
+        Mockito.when(userModel.getState()).thenReturn(state);
+        Mockito.when(userModel.getSubjects()).thenReturn(new ModelsListEmpty());
+        return userModel;
+    }
+
+    public void assertEquals(UserEntity first, UserEntity second) {
+        assertEquals(first, second, false);
+    }
+
+    public void assertEqualsWithoutId(UserEntity first, UserEntity second) {
+        assertEquals(first, second, true);
+    }
+
+    private void assertEquals(UserEntity first, UserEntity second, Boolean exceptId) {
+        if (!exceptId) {
+            Assert.assertEquals(first.getId(), second.getId());
+        }
+        Assert.assertEquals(first.getFirstName(), second.getFirstName());
+        Assert.assertEquals(first.getLastName(), second.getLastName());
+        Assert.assertEquals(first.getSurname(), second.getSurname());
+        Assert.assertEquals(first.getEmail(), second.getEmail());
+        Assert.assertEquals(first.getPassword(), second.getPassword());
+        Assert.assertEquals(first.getIsAdmin(), second.getIsAdmin());
+        Assert.assertEquals(first.getState(), second.getState());
+    }
+
+    public void assertNotEquals(UserEntity first, UserEntity second) {
+        assertNotEquals(first, second, false);
+    }
+
+    public void assertNotEqualsWithoutId(UserEntity first, UserEntity second) {
+        assertNotEquals(first, second, true);
+    }
+
+    private void assertNotEquals(UserEntity first, UserEntity second, Boolean exceptId) {
+        if (!exceptId) {
+            Assert.assertNotEquals(first.getId(), second.getId());
+        }
+        Assert.assertNotEquals(first.getFirstName(), second.getFirstName());
+        Assert.assertNotEquals(first.getLastName(), second.getLastName());
+        Assert.assertNotEquals(first.getSurname(), second.getSurname());
+        Assert.assertNotEquals(first.getEmail(), second.getEmail());
+        Assert.assertNotEquals(first.getPassword(), second.getPassword());
+        Assert.assertNotEquals(first.getIsAdmin(), second.getIsAdmin());
```

Comment:
Should we check 2 different users in NotEquals method using isAdmin for checking? What if we want to check 2 different users which are the same not the admins (or the are) that they are really 2 different person? (I have the same problem with NotEquals method in Topics hierarchy: we can have 2 different topics and they can have an identical field value subjectId because of 1:m sheme of Subject-Topic)

Reply:
@Ielay 
We should check that all fields are updated correctly. To be sure, first we ensure that all current values differs from new ones.
After update, we check that all values are the same.
Catch that?

Reply:
@malinink 
I understood that, but i meant that it's works for our particular field values in fixtures and additionals. I meant that if we would had an User (which we want to update) with admin=false and anotherUser (that will be replaced in DB) with admin=false then method assertNotEquals would be crashed.

Reply:
@Ielay Yeep, have you any ideas how to achieve in both cases?

Reply:
@Ielay The main purpose of such asserting is tending to ensure that we really update all other fields in `SQL` query except `id`. I don't see any other solution right now. And It seems to me that `hashCode` couldn't help us here.